src/GetUserForIdCommand.java

```diff
@@ -0,0 +1,29 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public class GetUserForIdCommand implements Command {
+    private UsersBook usersBook;
+    private int id;
```

Comment:
why do you need id as Command Field ? As I can see it uses only inside method exucute()